packages/cloud2edge/Chart.yaml

```diff
@@ -11,8 +11,8 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 apiVersion: v1
-version: 0.0.8
-appVersion: 0.0.8
+version: 0.0.9
+appVersion: 0.0.9
```

Comment:
this will need to be updated to 0.0.10 with the latest merge ...

Reply:
done